scripts/Logistics-Pipes.zs

```diff
@@ -843,8 +848,45 @@ recipes.addShaped(DisconnectWest, [
 [<ore:craftingToolScrewdriver>, DisconnectUp, null],
 [null, null, null]]);
 
+
 // ||||| Fluid Pipes |||||
-//Disabled for right now!
+Assembler.addRecipe(<LogisticsPipes:item.logisticsFluidContainer>, [<miscutils:gt.Volumetric_Flask_8k>, <ore:circuitBasic>, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.lead> * 144, 1600, 120);
+Assembler.addRecipe(<LogisticsPipes:item.logisticsFluidContainer>, [<miscutils:gt.Volumetric_Flask_8k>, <ore:circuitBasic>, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.tin> * 72, 1600, 120);
+Assembler.addRecipe(<LogisticsPipes:item.logisticsFluidContainer>, [<miscutils:gt.Volumetric_Flask_8k>, <ore:circuitBasic>, <gregtech:gt.integrated_circuit:18> * 0], <liquid:molten.solderingalloy> * 36, 1600, 120);
+
+
+// --- Basic Fluid Pipe
+Assembler.addRecipe(<LogisticsPipes:item.PipeFluidBasic>, [<LogisticsPipes:item.PipeItemsBasicLogistics>, <LogisticsPipes:item.logisticsFluidContainer>, <gregtech:gt.metaitem.01:32611>, <gregtech:gt.metaitem.01:32651>], <liquid:molten.lead> * 144, 1600, 120);

```

Comment:
Robot arm is a bit weird for something that is supposed to handle fluid only. Robot arm require 3 motor and 1 circuit to build. This would raise the price of each fluid pipe to 4 motor and 1 circuit, which even isn't the supplier/provider/extractor/insertion pipe you'd want to use yet.  I'd say `basic pipe + <logistic fluid container replacement> + pump` is enough.

Reply:
Removed the robot arm